csharp/test/Drivers/Interop/FlightSql/FlightSqlData.cs

```diff
@@ -204,5 +206,52 @@ private static SampleDataBuilder GetSQLiteSampleData()
 
             return sampleDataBuilder;
         }
+
+        private static SampleDataBuilder GetSpiceAISampleData()
+        {
+            SampleDataBuilder sampleDataBuilder = new SampleDataBuilder();
+
+            sampleDataBuilder.Samples.Add(
+
+                // Primitive types
+                new SampleData()
```

Comment:
Initial version includes basic/primitive types only. There will be separate PR to include more complex types